CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+- Check for negative day length in `Temporal.ZonedDateTime.prototype.round`. ([0d2d60ec], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/2261))
+- Reject Z designators when parsing `Temporal.PlainTime` strings.
+  This change avoids a common bug where a UTC timestamp is accidentally interpreted as if it were a local time in a real time zone.
+  If you do want to parse the time portion of a UTC timestamp string, use: `Temporal.Instant.from(s).toZonedDateTimeISO('UTC').toPlainTime()`. ([a7a50eac])
+- Reject 0-value components when parsing `Temporal.Duration` strings, and avoid rounding errors when nanoseconds components are present. ([58b5601a])
+- Reject `relativeTo` string options that are neither valid `Temporal.PlainDate` nor `Temporal.ZonedDateTime` strings, such as `'2022-08-18T17:01Z'`. ([4db15c41])
+- Add validation for the return values from calendar operations. ([d88cfa4d])
+- Validate required methods of `Temporal.TimeZoneProtocol` and `Temporal.CalendarProtocol`. ([84563cea], [755c7620], see also [Discussion](https://github.com/tc39/proposal-temporal/issues/2104#issuecomment-140))
+- Throw earlier when users might mix up positional `Temporal.TimeZone` and `Temporal.Calendar` arguments with each other, to prevent bugs like `new Temporal.ZonedDateTime(0n, cal, tz)` where the switched calendar and time zone arguments would cause exceptions to be thrown later. ([7922f1f9])
+
+Non-breaking changes:
+
+- Support new Internet Extended Date-Time (IXDTF) Annotations
+  - See the [Temporal + IXDTF tracking issue](https://github.com/tc39/proposal-temporal/issues/1450).
+  - Align ISO 8601 grammar with annotations from IXDTF specification.
+    Calendar and time zone annotations are now allowed to contain a "critical" flag (`'!'`) prefix.
+    Critical flags have no effect when parsing input strings because Temporal already treats unknown or inconsistent inputs as errors by default. ([e8b2e71c])
+    - There can be multiple annotations in a string, where unknown type annotations are ignored unless marked as critical.
+  - Allow `toString()` methods for `Temporal.PlainDate`, `Temporal.PlainDateTime`, `Temporal.PlainYearMonth`, `Temporal.PlainMonthDay`, and `Temporal.ZonedDateTime` to emit critical IXDTF annotations using the "critical" option.
+    Use this option with care, because the platform that you're communicating with may not understand this syntax.
+    `calendarName: 'critical'` behaves like `calendarName: 'always'` and `timeZoneName: 'critical'` behaves like `timeZoneName: 'always'`, but they also output a `'!'` flag in the corresponding annotation. ([50a64f16])
+    Critical flags are never used by Temporal, but could be consumed by other programs.
+  - Ignore calendar annotations when parsing `Temporal.Instant` strings. ([b86b87f0])
+  - Allow IXDTF annotations after YYYY-MM and MM-DD strings. UTC offsets in these annotations are forbidden and will cause an error to be thrown ([acd6464f])
```

Comment:
```suggestion
  - Allow calendar and/or time zone annotations after strings without a time part: YYYY-MM-DD, YYYY-MM, and MM-DD. ([acd6464f])
  - Disallow UTC offsets in YYYY-MM and MM-DD strings because they could cause ambiguity between an offset and the day of a YYYY-MM-DD string. ([acd6464f])
```
I think it's clearer if we split them.